.pre-commit-config.yaml

```diff
@@ -5,6 +5,8 @@ repos:
       - id: end-of-file-fixer
       - id: trailing-whitespace
       - id: detect-private-key
+      - id: no-commit-to-branch
```

Comment:
you can rm this one, there is branch protection on Github (on pushes on main)

Reply:
It's for local protection